libs/ledger-live-common/src/families/algorand/setup.ts

```diff
@@ -3,20 +3,16 @@
 import { createBridges } from "@ledgerhq/coin-algorand/bridge/js";
 import makeCliTools from "@ledgerhq/coin-algorand/cli-transaction";
 import algorandResolver from "@ledgerhq/coin-algorand/hw-getAddress";
-import { AlgorandAccount, Transaction, TransactionStatus } from "@ledgerhq/coin-algorand/types";
 import Algorand from "@ledgerhq/hw-app-algorand";
 import Transport from "@ledgerhq/hw-transport";
-import type { Bridge } from "@ledgerhq/types-live";
 import { CreateSigner, createResolver, executeWithSigner } from "../../bridge/setup";
 import type { Resolver } from "../../hw/getAddress/types";
 
 const createSigner: CreateSigner<Algorand> = (transport: Transport) => {
   return new Algorand(transport);
 };
 
-const bridge: Bridge<Transaction, AlgorandAccount, TransactionStatus> = createBridges(
```

Comment:
I think we still have a benefit to keep that type here to make sure we export what's expected, wdyt ?

Reply:
Why not :)